package mrrocpp.edp.sensor;

import "Common.proto";

message GenForceReading {
    required sint32 fx = 1;
    required sint32 fy = 2;
    required sint32 fz = 3;
    required sint32 tx = 4;
    required sint32 ty = 5;
    required sint32 tz = 6;
}

message SendBiasRequest {
    optional bool dummy = 1;
}

message SendBiasReply {
    optional bool dummy = 1;
}

service Ati3084 {
    rpc GetGenGForceReading (kiper.VoidMsg) returns (GenForceReading);
    rpc SendBias(kiper.VoidMsg) returns (SendBiasReply);
}
